examples/l3out/comprehensive/complete_syntax/policies.tf

```diff
@@ -0,0 +1,129 @@
+resource "aci_bfd_interface_policy" "bfd" {
+  tenant_dn     = aci_tenant.tenant.id
+  name          = "bfd_policy"
+  admin_st      = "enabled"
+  ctrl          = "opt-subif"
+  detect_mult   = "3"
+  echo_admin_st = "disabled"
+  echo_rx_intvl = "50"
+  min_rx_intvl  = "50"
+  min_tx_intvl  = "50"
+}
+
+resource "aci_rest_managed" "bfd_multihop_protocol_profile" {
+  dn         = "${aci_tenant.tenant.id}/bfdMhNodePol-test"
+  class_name = "bfdMhNodePol"
+  content = {
+    name       = "test"
+    adminSt    = "enabled"
+    minRxIntvl = "250"
+    minTxIntvl = "250"
+    detectMult = "3"
+  }
+}
+
+resource "aci_rest_managed" "bfd_multihop_interface_profile1" {
+  dn         = "${aci_tenant.tenant.id}/bfdMhIfPol-test1"
+  class_name = "bfdMhIfPol"
+  content = {
+    name       = "test1"
+    adminSt    = "enabled"
+    minRxIntvl = "250"
+    minTxIntvl = "250"
+    detectMult = "3"
+  }
+}
+
+resource "aci_rest_managed" "bfd_multihop_interface_profile2" {
+  dn         = "${aci_tenant.tenant.id}/bfdMhIfPol-test2"
+  class_name = "bfdMhIfPol"
+  content = {
+    name       = "test2"
+    adminSt    = "disabled"
+    minRxIntvl = "250"
+    minTxIntvl = "250"
+    detectMult = "3"
+  }
+}
```

Comment:
I have added this (bfdMhIfPol) resource in my current PR

Reply:
I'll include it after it is merged